nomad-base/src/contract_sync/mod.rs

```diff
@@ -46,12 +48,14 @@ where
 
 impl<I> ContractSync<I> {
     /// Instantiate new ContractSync
+    #[allow(clippy::too_many_arguments)]
```

Comment:
ouch 💀  put this on our tech debt plate

Reply:
https://github.com/nomad-xyz/rust/issues/44